oda_api/api.py

```diff
@@ -191,13 +188,20 @@ def __init__(self,
             if host.startswith('http'):
                 self.url = host
             else:
-                if protocol == 'http':
-                    self.url = "http://" + host
-                elif protocol == 'https':
-                    self.url = "https://" + host
-                else:
+                if protocol != 'http' and protocol != 'https':
```

Comment:
it is, surely, better, but we should really progress to discard this part, not improve :)

Reply:
Since I didn't know the real reason why `host` and `url` were both still available as parameter, I didn't want to, for the moment, change too much. 

But sure, it has to be improved

Reply:
Do we want still want to have the `host` parameter available? Is it still necessary?

Reply:
> The warning is already given, see there. Next step to discard. I think we could even make a more severe warning. It costs little to maintain current behavior.

So I will leave for now the `host` parameter compatibility.

> 
> But the url case should be implemented well, as it's the good way.

What is missing  to it?

Reply:
now a deprecation warning is sent to the user, with a recommendation to change the implementation